app/javascript/app/providers/climate-plans-provider/climate-plans-provider.js

```diff
@@ -0,0 +1,36 @@
+import { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getProvince } from 'selectors/provinces-selectors';
+import isEqual from 'lodash/isEqual';
+
+import * as actions from './climate-plans-provider-actions';
+import reducers, { initialState } from './climate-plans-provider-reducers';
+
+class ClimatePlansProvider extends PureComponent {
```

Comment:
I would just use `LocalizedProvider` here. Take a look at other providers, you can use `params` instead of `province` prop.

Reply:
👍 thanks for catching that